waiter/src/waiter/scheduler/marathon.clj

```diff
@@ -268,11 +268,12 @@
 
 (defrecord MarathonScheduler [marathon-api mesos-api retrieve-framework-id-fn
                               home-path-prefix service-id->failed-instances-transient-store
-                              service-id->kill-info-store force-kill-after-ms is-waiter-app?-fn]
+                              service-id->kill-info-store service-id->service-description
```

Comment:
Nice how the name just fits in and doesn't trigger downstream changes when removing the parameter :)